Utils/Dataflow/pyDKB/common/misc.py

```diff
@@ -0,0 +1,49 @@
+"""
+pyDKB.common.misc
+
+Miscellanious utility functions.
+"""
+
+import sys
+import inspect
+from datetime import datetime
+
+from types import logLevel
+
+# Datetime format for log messages
+DTFORMAT = '%Y-%m-%d %H:%M:%S'
+
+
+def log(message, level=logLevel.INFO, *args):
+    """ Output log message with given log level.
+
+    :param message: message to output
+    :type message: str
```

Comment:
By current function logic it's not `str`, it's `str` or `list` of anything. After resolution of https://github.com/PanDAWMS/dkb/pull/274/files#r317591401 this can probably change to `anything`.

Reply:
Changed in 5ada0e6f623391d551790ef59a94f89a62d9eb7c